common/src/test/java/io/pravega/common/concurrent/ConcurrentDependentProcessorTests.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.concurrent;
+
+import io.pravega.common.ObjectClosedException;
+import io.pravega.test.common.AssertExtensions;
+import io.pravega.test.common.ThreadPooledTestSuite;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
+import lombok.Cleanup;
+import lombok.val;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for the ConcurrentDependentProcessor class.
+ */
+public class ConcurrentDependentProcessorTests extends ThreadPooledTestSuite {
+    private static final int SHORT_TIMEOUT_MILLIS = 50;
+    private static final int TIMEOUT_MILLIS = 10000;
```

Comment:
This is not actually setting the test timeout. In fact I don't see one.

Reply:
fixed